src/app/core/services/auth.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+
+const storageKey = 'trainingPortalUser';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
```

Comment:
It is ok for now, but assuming later development it is better to use observables as returning result for almost all methods